yarn.lock

```diff
@@ -85,7 +85,7 @@ __metadata:
   languageName: node
```

Comment:
do we really need to update the root `yarn.lock`?

Reply:
Yes. We are using yarn workspaces and the only `yarn.lock` we have is in root of the project.